### PR TITLE
added new folder for multi gpu tensorflow example

### PR DESCRIPTION
**Pull request by ms4952**
Created a new folder: tensorflow-multigpu 
changed json to have free machine size
Added note for multi gpu example: this only runs on Pro and Enterprise
tensorflow-multigpu is not in the templates-hosted.json

### CODE REVIEW

File: examples/tensorflow-multigpu/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# TensorFlow 
+## Note: This example only runs on Pro and Enterprise
```
Comment (jnolis) [inline examples/tensorflow-multigpu/README.md]:
```suggestion

**Note: Due to the large hardware required for this example, it only runs on Saturn Cloud Pro and Saturn Cloud Enterprise.**
```

File: examples/tensorflow-multigpu/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+This resource gives you example for running TensorFlow training tasks on multiple GPU inside Saturn Cloud. 
+
+[multi-gpu-tensorflow](multi-gpu-tensorflow.ipynb) train your model with TensorFlow on multiple GPUs.
```
Comment (jnolis) [inline examples/tensorflow-multigpu/README.md]:
```suggestion
This is an example for running TensorFlow training tasks on multiple GPUs inside of Saturn Cloud. It uses a single large resource with many GPUs attach to concurrently train a model. The example is contained in [multi-gpu-tensorflow.ipynb](multi-gpu-tensorflow.ipynb).

To train a model with a single GPU in TensorFlow, see the single GPU TensorFlow example on the Saturn Cloud docs.
```

I didn't link to the single GPU example here, but we'll want to update it once the docs are updated

File: examples/tensorflow-multigpu/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+[multi-gpu-tensorflow](multi-gpu-tensorflow.ipynb) train your model with TensorFlow on multiple GPUs. 
+
+
+
+## Extra Resources
+* [Multi-GPU TensorFlow on Saturn Cloud](https://saturncloud.io/blog/tensorflow_intro/)
+* [Overview on GPUs](https://saturncloud.io/docs/reference/intro_to_gpu/)
+* [Learn more about TensorFlow](https://www.tensorflow.org/)
+* Prefer to use PyTorch? [Visit our docs about that!](https://saturncloud.io/docs/examples/pytorch/)
```
Comment (jnolis) [inline examples/tensorflow-multigpu/README.md]:
```suggestion
## Extra Resources
* [Overview on GPUs](https://saturncloud.io/docs/reference/intro_to_gpu/)
* [Learn more about TensorFlow](https://www.tensorflow.org/)
* [PyTorch Saturn Cloud example](https://saturncloud.io/docs/examples/python/pytorch/)
* [R TensorFlow Saturn Cloud example](https://saturncloud.io/docs/examples/r/torch/)
```

File: examples/tensorflow/README.md [hunk 4]
```diff
@@ -4,8 +4,6 @@ This resource gives you a starting example for running TensorFlow training tasks
 
 [single-gpu-tensorflow](single-gpu-tensorflow.ipynb) train your model with TensorFlow on a single GPU.
```
Comment (jnolis) [inline examples/tensorflow/README.md]: I would remove the reference to the notebook since it's the only one

File: examples/tensorflow/README.md [hunk 5]
```diff
@@ -4,8 +4,6 @@ This resource gives you a starting example for running TensorFlow training tasks
 
 [single-gpu-tensorflow](single-gpu-tensorflow.ipynb) train your model with TensorFlow on a single GPU. 
 
-[multi-gpu-tensorflow](multi-gpu-tensorflow.ipynb) train your model with TensorFlow on multiple GPUs. 
-
 
 
 ## Extra Resources
```
Comment (jnolis) [inline examples/tensorflow/README.md]: You should have the extra resources link to the doc page on multigpu saturn cloud, once that exists.

**Review by jnolis — APPROVED** [review]
Approved, after you make the suggestions I included.

File: examples/tensorflow/README.md (hunk 5 above)
Comment (ms4952) [reply examples/tensorflow/README.md]: Will add this , once I merge this pr and create docs for these examples .

**Pull request merged**